config/credentials/test.yml.enc

```diff
@@ -0,0 +1 @@
+zbSu6roXHfeKrMVuBsjaR461T8Su+8V1Tj/uE/oP5rB4ofsyHvF7WVL3iIidm+LNTK9bDtcd3TscDX4HCZyeL41vh155AQ==--0AlW/No5CR0Zj4Wb--BAL4jDBQEPau5mVgOeYj2g==
```

Comment:
Are you sure you want to commit these files?

Reply:
This configuration was created from `rails credentials: edit --environment [environment]` . At the moment of execution it generates two files with .key (which should not be uploaded to the repository) and then .yml.enc that if it allows me to upload to the repository